### PR TITLE
update UI

### PR DESCRIPTION
**Pull request by nguyenvanhieu97**
https://edu-redmine.framgia.vn/issues/4414

### CODE REVIEW

**Comment by nguyenvanhieu97** [comment]
pull này e có 26 files resource ạ

**Comment by nguyenvanhieu97** [comment]
ready

File: app/src/main/java/com/cris/nvh/framgiaproject/screen/home/HomeFragment.java [hunk 1]
```diff
@@ -190,9 +190,8 @@ private void initGenreAdapter() {
 	}
 
 	private void pageSwitcher(int time) {
-		Timer timer;
-		timer = new Timer();
-		timer.scheduleAtFixedRate(new RemindTask(), 0, time);
+		Timer timer = new Timer();
+		timer.schedule(new RemindTask(), 10, time);
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/cris/nvh/framgiaproject/screen/home/HomeFragment.java]: hardcode

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -14,11 +14,19 @@
         app:layout_constraintRight_toRightOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
+    <View
+        android:layout_width="0dp"
+        android:layout_height="1dp"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_main.xml]: hardcode

**Comment by NguyenMinhTuanB** [comment]
commented

**Comment by nguyenvanhieu97** [comment]
ready

**Pull request merged**